Dockerfile

```diff
@@ -1,9 +1,19 @@
 FROM python:3.6.3-stretch
 
 RUN apt-get -y update \
- && apt-get -y install wget unzip build-essential cmake pkg-config libatlas-base-dev gfortran \
-                       libgtk2.0-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg-dev libpng-dev \
-                       libtiff-dev libv4l-dev \
+  && apt-get -y install autoconf-archive automake g++ libtool pkg-config wget unzip build-essential cmake \
+     libatlas-base-dev gfortran libgtk2.0-dev libavcodec-dev libavformat-dev libswscale-dev libjpeg-dev libpng-dev \
+     libtiff-dev libv4l-dev libleptonica-dev \
+  && curl -sSL https://github.com/tesseract-ocr/tesseract/archive/ebbfc3ae8df85c351002000a76900e3086375e7b.zip -o tesseract-ocr.zip \
+  && unzip tesseract-ocr.zip \
```

Comment:
Need to clean up tesseract-ocr.zip and tesseract-${SHA} once we are done with them